book/src/snapshot-controller.md

```diff
@@ -0,0 +1,51 @@
+# Snapshot Controller
+
+## Status and Releases
+
+**Git Repository:** [https://github.com/kubernetes-csi/external-snapshotter](https://github.com/kubernetes-csi/external-snapshotter)
+
+**Status:** Beta
+
+### Snapshot Controller
+
+When Volume Snapshot is promoted to Beta in Kubernetes 1.17, the CSI external-snapshotter sidecar controller is split into two controllers: a snapshot-controller and a CSI external-snapshotter sidecar. See the following table for snapshot-controller release information.
+
+Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image | Min K8s Version | Max K8s Version | Recommended K8s Version
+--|--|--|--|--|--|--|--
+[external-snapshotter v2.0.0](https://github.com/kubernetes-csi/external-snapshotter/releases/tag/v2.0.0) | [release-2.0](https://github.com/kubernetes-csi/external-snapshotter/tree/release-2.0) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | quay.io/k8scsi/snapshot-controller:v2.0.0 | v1.17 | - | v1.17
+
+For more information on the beta version of the CSI external-snapshotter sidecar, see [this external-snapshotter page](external-snapshotter.md).
+
+## Description
+
+In the Beta version, the snapshot controller will be watching the Kubernetes API server for `VolumeSnapshot` and `VolumeSnapshotContent` CRD objects. The CSI `external-snapshotter` sidecar only watches the Kubernetes API server for `VolumeSnapshotContent` CRD objects.
```

Comment:
Do we want to mention something about it creating VolumeSnapshotContent objects to actuate the sidecar for provisioning?

Reply:
Sure.  I added one line there.  Details are in the snapshot KEP.